packages/graphql/tests/integration/issues/1050.int.test.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {graphql, GraphQLSchema} from "graphql";
```

Comment:
I noticed that your editor is not following our prettier style guidelines. This is on us as it should be done automatically.

If you could please run the following command and update this PR:
```sh
npx prettier packages/graphql/tests/integration/issues/1050.int.test.ts packages/graphql/src/translate/create-projection-and-params.ts --write
```
Sorry for the inconvenience :pray:

Reply:
no problem, thanks for the quick review!